Tests/TestUser.php

```diff
@@ -43,7 +43,7 @@ public function getPassword()
      */
     public function getSalt()
     {
-        return null;
```

Comment:
OK this was fixed by php-cs-fixer with "old" parameters from ci. I've seen you changed the settings little bit :-)